packages/bento-design-system/src/Charts/BarChart/BarChart.tsx

```diff
@@ -63,21 +63,21 @@ export function BarChart<D extends string, C extends string>({
       debounce={debounce}
     >
       <RechartBarChart data={data}>
-        {showXAxis && <XAxis dataKey={dataKey} />}
-        {showYAxis && <YAxis />}
-        {showTooltip && (
+        {hideXAxis && <XAxis dataKey={dataKey} />}
```

Comment:
shouldn't you invert the condition here?
!hideXAxis && <...

Reply:
damn, I was multitasking too much :D